debug/testlib.py

```diff
@@ -142,9 +142,10 @@ def wait(self, *args, **kwargs):
         return self.process.wait(*args, **kwargs)
 
 class VcsSim(object):
-    logname = "simv.log"
+    logfile = tempfile.NamedTemporaryFile(prefix='simv', suffix='.log')
+    logname = logfile.name
 
-    def __init__(self, sim_cmd=None, debug=False):
+    def __init__(self, sim_cmd=None, debug=False, timeout=300):
```

Comment:
is there a path to override this timeout? Let's not hard-code any timeouts.

Reply:
For example can target class have a "init_timeout" variable which we pass in here but can be overridden in the target subclass?

Reply:
Yes. The function is called from the .py file containing a target configuration, so each target can specify its own timeout.